geode-wan/src/test/resources/expected-pom.xml

```diff
@@ -71,6 +71,11 @@
       <artifactId>geode-core</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.geode</groupId>
+      <artifactId>geode-cq</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
If the dependency from `geode-cq` is moved to `geode-dunit/build.gradle` this file needs not be updated. Instead a different  `expected-pom.xml` must be updated.